numba/ir_utils.py

```diff
@@ -971,6 +972,24 @@ def rename_labels(blocks):
 
 def simplify_CFG(blocks):
     """transform chains of blocks that have no loop into a single block"""
+    # first, inline single-branch-block to its predecessors
```

Comment:
Resolved. Paul says this comes from user `assert()` calls with multiple conditions.